docs/Use Cases/How To Set Up HSM Delivery Dashboard.md

```
@@ -7,7 +7,7 @@ Context: Meta recently introduced something called as Frequency capping. Explain
 
 To prevent misuse and gaming the system, they haven’t told us how many messages are allowed in how many days. You can read more about this in their own words [here.](https://www.gupshup.io/resources/blog/all-you-need-to-know-about-whatsapp-business-api-frequency-capping)
 
-### Solution: To understand contacts, templates, and campaigns (based on dates the broadcast was initaited) by having this particular information visialized on a lookerstudio dashboard. This will reduce the effort in repeteadly quering this infomration from the database for every braodcast made, and to make this information easily accessible to the program / operations team to work with.
+### Solution: To understand contacts, templates, and campaigns affected (based on dates the broadcast was initaited) by having information of errors that were encountered visialized on a lookerstudio dashboard. This will reduce the effort in repeteadly quering this infomration from the database for every braodcast made, and to make this information easily accessible to the program / operations team to work with.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)